### PR TITLE
Add level parameter to inventory.write stationXML

### PR DESCRIPTION
**Pull request by Jollyfant**
Adds kwarg `level` for Inventory.write for StationXML. `FDSNWS-Station` supports these four levels and I wanted to try to use ObsPy running behind FDSNWS Station writing at different levels. Maybe it's useful.

### CODE REVIEW

**Comment by krischer** [comment]
Yea - would be very useful. This discussion here is also relevant: https://github.com/krischer/jane/issues/33

**Comment by krischer** [comment]
Still requires a couple basic tests.

File: obspy/io/stationxml/core.py [hunk 1]
```diff
@@ -825,8 +825,12 @@ def _write_stationxml(inventory, file_or_file_object, validate=False,
 
     etree.SubElement(root, "Created").text = _format_time(inventory.created)
 
+    level = kwargs.get("level") or "response"
```
Comment (megies) [inline obspy/io/stationxml/core.py]: could also do `kwargs.get("level", "response")`

**Comment by megies** [comment]
Maybe this kwarg could also be added to `Inventory/Network.select()`, but maybe it's not so important there..

File: obspy/io/stationxml/core.py (hunk 1 above)
Comment (Jollyfant) [inline obspy/io/stationxml/core.py]: Neat

**Comment by Jollyfant** [comment]
Added a basic test. Is Jane actively undergoing development?

**Comment by krischer** [comment]
> Is Jane actively undergoing development?

It works and crucial bugs are fixed but currently there are no large efforts to add new features whatever these may be. We are definitely also open to pull and feature requests.

File: obspy/io/stationxml/core.py [hunk 2]
```diff
@@ -825,8 +825,12 @@ def _write_stationxml(inventory, file_or_file_object, validate=False,
 
     etree.SubElement(root, "Created").text = _format_time(inventory.created)
 
+    level = kwargs.get("level", "response")
```
Comment (krischer) [inline obspy/io/stationxml/core.py]: Please make this a proper argument to `_write_stationxml()`.

**Comment by megies** [comment]
Do we want a changelog for this?

**Comment by Jollyfant** [comment]
Added to change log.

**Comment by krischer** [comment]
Looks pretty done to me. Can this be merged? Thanks a lot!

**Pull request merged**